src/app/app.component.ts

```diff
@@ -290,18 +290,28 @@ export class AppComponent implements OnInit, AfterViewInit {
                     options.headers.set('Authorization', this.Signature);
                     options.headers.set('X-Auth-Date', this.kDate);
                     this.http.put("/"+ uploadUrl + '?partNumber=' + (i + 1) + '&uploadId=' + uploadId, chunk, options).subscribe((data) => {
-                        let x2js = new X2JS();
-                        let jsonObj = x2js.xml_str2json(data['_body']);
-                        window['uploadPartArr'].push(jsonObj);
+                        let header = data.headers['_headers']
```

Comment:
Since you are changing the current upload functionality it is very important that it is tested thoroughly. Have you tested this with the latest installation of OpenSDS?

Reply:
> Since you are changing the current upload functionality it is very important that it is tested thoroughly. Have you tested this with the latest installation of OpenSDS?

Yes，we have